bin/server.js

```diff
@@ -1,16 +1,43 @@
 var mongoose    = require('mongoose');
 var express     = require('express');
 var app         = express();
+var bodyParser  = require('body-parser');
 var mongoURI    = process.env.MONGO || 'mongodb://localhost:27017/nodeadvca';
+var routes      = require('../lib/routes');
 var log         = require('../middleware/logger').child({
   level : 'info',
   component : '[BIN-SERVER]'
 });
 
-// set port for server
-var port = process.env.PORT || 3000;
+function instantiateServer() {
+	// configure app to use bodyParser() this will let us get the data from a POST & PUT requests
+	app.use(bodyParser.urlencoded({ extended: true }));
+	app.use(bodyParser.json());
+
+	// set port for server
+	var port = process.env.PORT || 3000;
+
+	app.use(routes);
```

Comment:
this is great and will work ... just to put a doubt in your head have a rad through comment further down referencing this